armbian/base/config/redis/factorysettings.txt

```diff
@@ -14,9 +14,10 @@ SET tor:ssh:enabled 0
 SET tor:electrs:enabled 1
 SET tor:bbbmiddleware:enabled 1
 
-SET bitcoind:version xxx      
+SET bitcoind:version xxx
 SET bitcoind:ibd 1
 SET bitcoind:ibd-clearnet 0
+SET bitcoind:assumevalid 1
```

Comment:
IIRC Bitcoin Core's `assumevalid` takes a block-hash which is assumed to be valid (i.e. only check the proof of work, no tx signatures before the given block). The current default value is defined [here](https://github.com/bitcoin/bitcoin/blob/00ffe5aca134f424351b05a85da4eef9c3e4553a/src/chainparams.cpp#L97) (Block 563378). I don't see the reason to change the default `assumevalid`.

(is the 1 a placeholder? if yes it should probably be `xxx`)

Reply:
Good find, I was under the impression it's either turned on by default (1) or off. But it's not working like that, I'll remove this change for now.